### PR TITLE
.net standard support

### PR DESCRIPTION
**Pull request by claudiuslollarius**
Updated solution

- No code changes
- Targeting .net standard instead of PCLs (base library is net standard 1.2)
- Updated other projects to use more recent SDKs, nuget dependencies
- Updated projects to use package references instead of package.config/project.json since they are deprecated

### CODE REVIEW

**Comment by albertofustinoni** [comment]
Is there a way to see Bitrise's build log? The PR builds on my system...

File: Platforms/NGraphics.iOS.Test/NGraphics.iOS.Test.csproj [hunk 1]
```diff
@@ -105,7 +105,9 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <ImageAsset Include="Resources\Images.xcassets\AppIcons.appiconset\Contents.json" />
+    <ImageAsset Include="Resources\Images.xcassets\AppIcons.appiconset\Contents.json">
+      <Visible>false</Visible>
```
[the remark this answers is not in the document]
Comment (claudiuslollarius) [reply Platforms/NGraphics.iOS.Test/NGraphics.iOS.Test.csproj]: This is done automatically by visual studio every time the project gets opened.
Does not affect behaviour as far as I can tell.

**Comment by claudiuslollarius** [comment]
Updated PR with fixes for the issues you have pointed out.

**Comment by praeclarum** [comment]
Thank you!

**Pull request merged**